protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -265,3 +265,38 @@ service ObservabilityService {
 
   rpc SendMetric(SendMetricRequest) returns (SendMetricResponse);
 }
+
+message MetricCounter {
+  int64 value = 1;
+}
+
+message MetricHistorgram {
```

Comment:
Oh interesting. If we're modelling these in proto anyway, I wonder if we should model model them in the DB too...🤔

Reply:
I think we could. One thought I had is that the console service might act like a BFF for the FE so I could see a world where these types are super specific to the UI vs. the db. I think those are basically the same now, but I guess they could be different in the future. Whatcha think?